### PR TITLE
Opinion on protected branch configuration

### PR DESCRIPTION
**Pull request by adborden**
Include an opinionated configuration for branch protection on our repositories.

### CODE REVIEW

File: how_we_work/standards.md [hunk 1]
```diff
@@ -19,6 +19,16 @@ Beyond the TTS-wide [language selection](https://engineering.18f.gov/language-se
 
 All the above are "where possible" — there will be exceptions.
 
+## Protected branches
+
+Protected branches are [required](https://engineering.18f.gov/project-setup/#branch-protection-requirement) for the `main` branch in GitHub. Additionally, we prefer this configuration for all of our repositories.
+
+- "Require pull request reviews before merging"
+- "Dismiss stale pull request approvals when new commits are pushed"
```
Comment (afeld) [inline how_we_work/standards.md:27]: I think this one is overkill, but don't feel _super_ strongly.

```suggestion
```

**Review by afeld — APPROVED** [review]
Will wait for @JJediny to review since this is a team policy change.

File: how_we_work/standards.md (hunk 1 above)
Comment (adborden) [reply how_we_work/standards.md:27]: Same. 🤷  wearing my government compliance hat, if we say everything is peer reviewed then we should enforce it. But it's not like we have an ATO or SSP that explicitly states that. Maybe it's a "best practice" example we want to set for other teams?

fwiw, I don't enable this option for my non-government projects and I trust the team to do the right thing.

**Comment by JJediny** [comment]
Terraform plan in __terraform__
<details>
  <summary>No changes. Infrastructure is up-to-date.</summary>

```
No changes. Infrastructure is up-to-date.

This means that Terraform did not detect any differences between your
configuration and real physical resources that exist. As a result, no
actions need to be performed.
```
</details>

Plan generated in [plan #7](https://github.com/18F/tts-tech-portfolio/actions/runs/969698503)

**Review by JJediny — APPROVED** [review]
I'm fine with these

**Pull request merged**